client/my-sites/stats/hooks/use-subscribers-query.tsx

```diff
@@ -1,53 +1,67 @@
-import { useQuery } from '@tanstack/react-query';
+import { useQuery, useQueries, UseQueryResult } from '@tanstack/react-query';
 import wpcom from 'calypso/lib/wp';
 
-interface SubscriberPayload {
+export interface SubscriberPayload {
 	date: string;
 	unit: string;
 	data: any[]; // TODO: add type
 	fields: string[];
 }
 
-export function querySubscribers(
+interface SubscribersData {
```

Comment:
```suggestion
export interface SubscribersData {
```

You can add an export directly to the interface declaration!